README.md

```diff
@@ -115,6 +118,39 @@ maven_jar(
 )
 ```
 
+### Source code formatting
```

Comment:
Let's hold off on documenting this stuff for now. I should probably look into what it would take to get us a proper documentation website set up. We're long overdue for that.